src/services/securityGroup/format.ts

```diff
@@ -72,7 +73,12 @@ export default ({
       ;(rule.ipRanges || []).map(
         ({ cidrIp, description: ipRangesDescription = '' }) => {
           allRules.push({
-            id: cuid(),
+            id: generateUniqueId({
+              arn,
+              cidrIp,
+              ipRangesDescription,
```

Comment:
I'm seeing some duplicate id errors:

![image](https://user-images.githubusercontent.com/64504276/172415134-3c008d50-2718-46f1-ae4f-5186f154a600.png)

It seems that `cidrIp `and `ipRangesDescription` can be the same between the rules, which would generate the same id in those cases.

Reply:
hmm I wonder why these are happening, unless the same rule is defined twice on the same arn 🤔

Reply:
Not exactly the same rule, but yes those 2 values that compose the Id. For example, the rules of this security group:

![image](https://user-images.githubusercontent.com/64504276/172439255-2fa0a23f-48a5-4c6a-93d1-a13904cb8261.png)

Reply:
hmm maybe we need to change up this obj structure in that case

Reply:
Yes, it should at least include all the properties that make the rule unique. On the other hand here I added a log to the generation of these ids to validate that this is happening:

![image](https://user-images.githubusercontent.com/64504276/172443092-cb129058-dcb5-4d6a-9913-65970d5e8c82.png)

![image](https://user-images.githubusercontent.com/64504276/172443206-4f648482-f47f-472d-962d-12849c4bee38.png)

Reply:
Hey @m-pizarro I improved the uniqueness of these nested objects. Please give it another try when you have a chance

Reply:
@mfranceschit yeah, everything looks good now, no more errors are logged. Thanks!